docs/sources/experimental/experimental.md

```diff
@@ -1,11 +1,21 @@
 page_title: Overview of Experimental Features
 page_keywords: experimental, Docker, feature
 
-# Overview 
+# Experimental Features in this Release 
 
-This page contains a list of features in the Docker engine which are experimental as of the current release. Experimental features are **not** ready for production.  They are provided to for test and evaluation in your sandbox environments.  
+This page contains a list of features in the Docker engine which are
+experimental as of the current release. Experimental features are **not** ready
+for production. They are provided for test and evaluation in your sandbox
+environments.  
 
-The information below describes each feature and the Github pull requests and issues associated with it. If necessary, links are provided to additional documentation on an issue.  As an active Docker user and community member, please feel free to provide any feedback on these features you wish.
+The information below describes each feature and the Github pull requests and
+issues associated with it. If necessary, links are provided to additional
+documentation on an issue.  As an active Docker user and community member,
+please feel free to provide any feedback on these features you wish.
+
+## Install Docker experimental 
+
+TODO: Will experimental be available on all platforms or just some?
```

Comment:
I can imagine that depends on the feature, but should be mentioned in the description.